orttraining/orttraining/models/bert/main.cc

```diff
@@ -67,6 +67,8 @@ Status ParseArguments(int argc, char* argv[], BertParameters& params, OrtParamet
         cxxopts::value<std::string>()->default_value(""))
       ("output_dir", "The output directory where the trained model files will be written.",
         cxxopts::value<std::string>()->default_value(""))
+      ("perf_output_dir", "The output directory where the trained perf metrics files will be written.",
```

Comment:
there's only one file, how about just specifying the output file path?

Reply:
let's discuss as I have one follow-up question on CI build.

Reply:
keep it the same per discussion